presto-jdbc/src/test/java/io/prestosql/jdbc/TestJdbcPreparedStatement.java

```diff
@@ -263,6 +266,47 @@ public void testExecuteUpdate()
         }
     }
 
+    @Test
+    public void testExecuteBatch()
+            throws Exception
+    {
+        try (Connection connection = createConnection("memory", "default")) {
+            try (Statement statement = connection.createStatement()) {
+                statement.execute("CREATE TABLE test_execute_batch (c_int integer)");
+            }
+
+            try (PreparedStatement preparedStatement = connection.prepareStatement(
+                    "INSERT INTO test_execute_batch VALUES (?)")) {
+                for (int i = 0; i < 3; i++) {
+                    preparedStatement.setInt(1, i);
+                    preparedStatement.addBatch();
+                }
+                assertEquals(preparedStatement.executeBatch(), new int[]{1, 1, 1});
+
+                try (Statement statement = connection.createStatement()) {
+                    ResultSet resultSet = statement.executeQuery("SELECT c_int FROM test_execute_batch ORDER BY 1");
+                    for (int i = 0; i < 3; i++) {
+                        assertTrue(resultSet.next());
+                        assertEquals(i, resultSet.getInt(1));
+                    }
+                    assertFalse(resultSet.next());
+                }
```

Comment:
You can express this more concisely  as in
https://github.com/prestosql/presto/blob/7771f046f19b868e1cde7e04876dd45d8b0647dd/presto-jdbc/src/test/java/io/prestosql/jdbc/TestPrestoDatabaseMetaData.java#L404

(and then you can remove ORDER BY)